style.css

```diff
@@ -23,41 +23,45 @@ body {
 h1,
 h2,
 h3 {
-    font-weight: 700;
+    font-family: Raleway, sans-serif;
```

Comment:
Přidal bych ještě nějaký fallback mezi to, Arial, Helvetica, možná Roboto, možná něco z Windowsích fontů.